consensus/src/chained_bft/event_processor.rs

```diff
@@ -801,20 +808,20 @@ impl<T: Payload> EventProcessor<T> {
             {
                 counters::CREATION_TO_COMMIT_S.observe_duration(time_to_commit);
             }
-            let compute_result = self
-                .block_store
-                .get_compute_result(committed.id())
-                .expect("Compute result of a pending block is unknown");
```

Comment:
I don't see how it's okay to remove the expect() here.  Except for the root block, all other blocks must have a compute_result.  If the expect fails in this method, it means that the BlockStore is no longer in a consistent state.

Reply:
Yes, indeed, the executed state is not inserted independently of the block and the proper potential asynchronous failure we want is here:
https://github.com/libra/libra/blob/146714289c8424aff0098650d22736cf5fc3466a/consensus/src/chained_bft/block_storage/block_store.rs#L194-L201
not in the present PR => I've removed that commit.